.nais/redis-config.yaml

```diff
@@ -23,3 +23,34 @@ spec:
       memory: 256Mi
   service:
     port: 6379
+
```

Comment:
Det må legges på accessPolicy her, men det kan vi kanskje vente med til vi migrerer til gcp.

Reply:
Ja, det trengs vel bare for GCP så like greit å vente.